### ISSUE TITLE
[Question] Nuki Opener Ring-To-Open with relay

### ISSUE THREAD

**Issue by to2tx** [kvj/hass_nuki_ng#136]
Hello,

I have been looking at the docs and other issues/questions asked but am not 100% sure my usecase would work as expected.

TLDR:

Can I use the Nuki-NG Home Assistant integration for the Nuki Opener with automations such that whenever I enter my home zone it will activate the Ring-To-Open function of Nuki Opener (hence without using the Nuki App). When I press the ring button I am not sure what events are triggered or states are changed for the Nuki Opener? E.g. would I be able to detect the button press via some event or state change that shows the button is pressed? Alternatively, will the Nuki Opener automatically turn off the Ring-To-Open mode with a single button press (this should also allow Home Assistant to understand that the button was pressed)?

Use-Case:

I am considering buying a Nuki Opener using a digital intercom that is actually not supported regarding opening of the door. The door in question is of course not an entrance to the house but a garden door at the street to access the property. Since I live in single-family house I have full access to the intercom and have installed a Shelly 1 relay that allows me to open the door remotely from Home Assistant.

What I want to understand is if I could use a Nuki Opener connected in such a way that its function only is to intercepts the signal ringing the door and using Home Assistant pressing the button triggers the Shelly 1 relay to open the door. In other words I want to have the Ring-To-Open functionality but which triggers the relay instead (or in addition) to sending the open signal to the intercom (as that regardless won't work).
